ignite/pkg/cosmosgen/cosmosgen.go

```diff
@@ -37,6 +37,10 @@ type generateOptions struct {
 }
 
 // TODO add WithInstall.
+type ModuleIncludes struct {
+	Includes []string
+	Modules  []module.Module
+}
```

Comment:
It would be helpful to have a documentation string to explain the role of this type:
```suggestion
// ModuleIncludes defines ...
type ModuleIncludes struct {
	Includes []string
	Modules  []module.Module
}
```

Reply:
WIll update as part of the other change you mentioned that sends thirdModuleIncludes to a different map